share/functions/__fish_print_portage_repository_paths.fish

```diff
@@ -1,4 +1,14 @@
 function __fish_print_portage_repository_paths --description 'Print the paths of all configured repositories'
-    # repos.conf may be a file or a directory
-    find /etc/portage/repos.conf -type f -exec cat '{}' + | string replace -r --filter '^\s*location\s*=\s*(\S+)' '$1'
+    set -l a /etc/portage/repos.conf
+    set -l b
+    set -l c /usr/share/portage/config/repos.conf
+    test -d $a
+    and set b (find $a -type f )
+    test -f $a
+    and set b $a
```

Comment:
I liked the if-else-if-chain you had before :(

Reply:
It's more readable with all the 'end's but as a returning user I felt some desire to use the new syntax, but have no strong opinion about it